### PR TITLE
feature/has-symbol

### PR DESCRIPTION
**Pull request by EdgarAyllon**
added feature/has-symbol
 added jasmine 2.9.1

### CODE REVIEW

**Comment by EdgarAyllon** [comment]
Removed camelCase conflict.

File: course/frontend/has-symbol/test/hassymbolSpecs.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+describe('has symbol test', function () {
```
Comment (VGamezz19) [inline course/frontend/has-symbol/test/hassymbolSpecs.js]: hey! please rename hassymbolSpect.js to  estandar name `hassimbol**.spec.**js`

File: course/frontend/has-symbol/scripts/has-symbol.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+//Function that detects if a string has any symbol (e.g. hasSymbol('hello%') => true).
+// RETURN true if any, otherwise false.
+'use strict'
+function hasSymbol( _string ){
+
+	if ( typeof _string !== 'string' ||	 _string.length === 0 || (/[\n\t]/g).test( _string) ) return 'ERROR: OUT of SPECS' ;
```
Comment (VGamezz19) [inline course/frontend/has-symbol/scripts/has-symbol.js]: Hello @EdgarAyllon, to control and create new errors, it is better to use `try catch`. In standard programming, the definition of variables is made immediately after declaring a function

File: course/frontend/has-symbol/test/hassymbolSpecs.js (hunk 1 above)
Comment (EdgarAyllon) [reply course/frontend/has-symbol/test/hassymbolSpecs.js]: Hi, thanks for the advice !!! but this is a little description, not the name itself.

File: course/frontend/has-symbol/scripts/has-symbol.js (hunk 2 above)
Comment (EdgarAyllon) [reply course/frontend/has-symbol/scripts/has-symbol.js]: Yes, I know, thanks. Note as been taked for the next challenge.

**Comment by manuelbarzi** [comment]
WELL done! this PR is ready to merge. ASK 3 other reviewers to take a look and confirm it (+1) and THEN we proceed to merge it ,)

**Comment by VGamezz19** [comment]
+1

**Pull request merged**